draft-ecahc-moderation.md

```diff
@@ -349,6 +349,10 @@ Potential abuse of the moderation process for the suppression of
 undesired opinions is counteracted by the availability of an appeals
 process, per {{appeals}}.
 
+The actions of the moderation team are intended to limit the likelihood
+of disruptive behaiviour by one IETF participant from discouraging
```

Comment:
```suggestion
of disruptive behavior by one IETF participant from discouraging
```
But is it really always one? There have been discussions where several participants went over the line, often in response to each other.

Reply:
Thanks fro fix the typo.  

s/by one IETF participant/by a few IETF participants/